### DISCUSSION TITLE
Summary of all available functions

### DISCUSSION THREAD

**Discussion by Roma004** [uNetworking/uWebSockets#1022] [Q&A] [answered]
Whether it is possible to make something like a summary of all classes and functions inherent in them and, preferably, with a minimal description of their use. As for me, is realy difficult to navigate among small incomplete examples, where not all the features of this library are disclosed.

**Comment by ghost** [comment, marked as answer]
Did you check out the JavaScript docs? They are 1-to-1 with the C++ base library and can be very helpful getting an overview; https://unetworking.github.io/uWebSockets.js/generated/

**Comment by Roma004** [comment]
I'm sorry, I didn't think to get in there, it was a stupid question! Thank you for explaining!

**Comment by ghost** [comment]
No problem, I am considering adding C++-specific docs.
